dali/base/dadfs.cpp

```diff
@@ -12045,8 +12062,8 @@ IDFAttributesIterator* CDistributedFileDirectory::getLogicalFilesSorted(
     }
     IArrayOf<IPropertyTree> results;
     Owned<IElementsPager> elementsPager = new CDFUPager(udesc, (const char*) filters, localFilters, (const char*) localFilterBuf,
-        so.length()?so.str():NULL );
-    getElementsPaged(elementsPager,startOffset,maxNum,NULL,"",cacheHint,results,total,false);
+        so.length()?so.str():NULL);
+    getElementsPaged(elementsPager,startOffset,maxNum,NULL,"",cacheHint,results,totalReturned,total,false);
```

Comment:
minor: Even though for DFU the pager doesn't return a connection, getElementsPaged()'s definition says it might, so it would be better to assign the result to an Owned<IRemoteConnection> here for sanity.